ezshare-frontend/src/App.jsx

```diff
@@ -268,6 +291,16 @@ const Browser = () => {
       </Section>
 
       <Section>
+        <form action="/api/zipfilesdownload" method="post" id="sendform">
```

Comment:
i don't think we need a form here

Reply:
okay will remove the form and follow the anchor tag method